package.json

```diff
@@ -45,9 +45,9 @@
     "estraverse": "^4.2.0",
     "esutils": "^2.0.2",
     "file-entry-cache": "^1.1.1",
-    "glob": "^7.0.3",
+    "glob2": "^7.0.3",
```

Comment:
Or maybe we can monkeypatch glob if the issue doesn't get accepted? It looks like there hasn't been any movement on the PR at all. Not even a note of any kind.